code/modules/clothing/shoes/magboots.dm

```diff
@@ -73,7 +73,10 @@
 	var/state = "disabled"
 	if(src.clothing_flags&NOSLIP)
 		state = "enabled"
-	to_chat(user, "<span class='info'>Its mag-pulse traction system appears to be [state].</span>")
+	if(base_state == "capboots")
```

Comment:
I think a better way to do this would be to give magboots something like an `var/anchor_system = "mag-pulse traction system"` and then override that var on the captain's boots to equal `"anchoring spikes"`.
Then the magboots' examine could simply use `to_chat(user, "<span class='info'>Its [anchor_system] appear\s to be [state].</span>")`.

Reply:
Or do like Exxion said, which I just noticed, and add complete strings for enabled/disabled text as vars to the magboots.